src/index.html

```diff
@@ -5,8 +5,383 @@
+        >
+
+        <h2 class="card__product">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__stars_reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-name">Price:</span>
+          <span class="card__price-value">$2,199</span>
+        </div>
+
+        <a
+          href="#home" 
+          class="card__button"
+        >
+          buy
+        </a>
+      </div>
+      <div class="card">
+        <img 
+          src="./images/imac.jpeg" 
+          class="card__image" 
+          alt="ProductImage"
+        >
+
+        <h2 class="card__product">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__stars_reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-name">Price:</span>
+          <span class="card__price-value">$2,199</span>
+        </div>
+
+        <a
+          href="#home" 
+          class="card__button"
+        >
+          buy
+        </a>
+      </div>
+      <div class="card">
+        <img 
+          src="./images/imac.jpeg" 
+          class="card__image" 
+          alt="ProductImage"
+        >
+
+        <h2 class="card__product">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__stars_reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-name">Price:</span>
+          <span class="card__price-value">$2,199</span>
+        </div>
+
+        <a
+          href="#home" 
+          class="card__button"
+        >
+          buy
+        </a>
+      </div>
+      <div class="card">
```

Comment:
Please fix everywhere
```suggestion
      </div>
      
      <div class="card">
```